cypress/integration/price/pricelist_with_currency_CHF_and_precision_3_spec.js

```diff
@@ -0,0 +1,29 @@
+import { PriceList } from '../../support/utils/pricelist';
+import { Pricesystem } from '../../support/utils/pricesystem';
+import { humanReadableNow } from '../../support/utils/utils';
+
+describe('Create Pricelist for Automatic End2End Tests with cypress https://github.com/metasfresh/metasfresh-e2e/issues/195', function() {
```

Comment:
I think the description here should also be `Create new PriceList with currency CHF and precision 3`, even though it's duplicated with the `it` test below.